ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -546,6 +547,7 @@ - (ORKSurveyAnswerCell *)answerCellForTableView:(UITableView *)tableView {
                                @(ORKQuestionTypeTimeInterval) : [ORKSurveyAnswerCellForPicker class],
                                @(ORKQuestionTypeInteger) : [ORKSurveyAnswerCellForNumber class],
                                @(ORKQuestionTypeEligibility) : [ORKSurveyAnswerCellForEligibility class]};
```

Comment:
Remove the `};` at the end of this line and replace with `,`. The project isn't compiling.